edx_proctoring/backends/rest.py

```diff
@@ -197,6 +199,11 @@ def on_review_callback(self, attempt, payload):
         Called when the reviewing 3rd party service posts back the results
         """
         # REST backends should convert the payload into the expected data structure
+        if payload.get('reviewed_by', False):
+            try:
+                payload['reviewed_by'] = User.objects.get(email=payload['reviewed_by'])
```

Comment:
We don't currently expose the email address, but the user id to our rest api implementers (via the JWT token). That's a hole in my diff as of right now.

Reply:
Update: I'm wrong; the JWT includes both the user's id as well as their email. Implementers should be able to provide this.